lighthouse-core/test/audits/seo/plugins-test.js

```diff
@@ -16,44 +16,52 @@ describe('SEO: Avoids plugins', () => {
       [{
         tagName: 'APPLET',
         params: [],
+        node: {},
```

Comment:
since this PR is adding a new `EmbeddedContent` artifact property and changing the audit details almost completely and we're trying to reach toward a typed test future (#11728), can these be generated so they're more like the real thing and the output tested to some degree?

Reply:
I'm struggling to find any actual pages with these elements. The one I found isn't included in our plugin list: https://www.quackit.com/html_5/tags/html_object_tag.cfm (`video/quicktime`). Should that type be included?

EDIT: well `video/quicktime` is expected to pass in a test. I don't understand the purpose of this audit if a quicktime plugin is acceptable. Is the idea that any video is OK? I suppose since this is an SEO audit video is out of scope, it's just that this audit also could double as a user/compat/browser-friendliness check.